components/tidb_query_datatype/src/codec/mysql/enums.rs

```diff
@@ -0,0 +1,89 @@
+use std::cmp::Ordering;
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Enum {
+    data: Vec<u8>,
+    offset: Vec<usize>,
```

Comment:
Can we use `BufferVec` for this part?

Reply:
Let me check it out first.

Reply:
Let's good to me, I will migrate to `BufferVec`